source/imaer-gml/src/test/java/nl/overheid/aerius/gml/GMLCalculationSetOptionsReaderTest.java

```diff
@@ -176,6 +178,25 @@ void testReadCalculationSetOptionsDuplicateOptions() {
     assertEquals("somewhere else", options.getNcaCalculationOptions().getPermitArea(), "PermitArea");
   }
 
+  @Test
+  void testOWN2000ReadCalculationSetOptions() {
```

Comment:
Should this not be `OwN` instead of `OWN`? That's the name I see used in files in this PR and in existing code